projects/plugins/social/readme.txt

```diff
@@ -104,9 +104,23 @@ The easiest way is to use the Custom Message option in the publishing options bo
 6. Managing Social media accounts in the post editor
 
 == Changelog ==
-### 6.3.0 - 2025-03-21
+### 6.4.0 - 2025-04-17
 #### Added
-- My Jetpack: An onboarding screen has been implemented to provide clear, step-by-step instructions for new users connecting to Jetpack.
+- Added more error logging.
+- Added Share status support for WPCOM sites
+- Adding My Jetpack tour.
+- Adds Account Protection initialization
+- Improve the onboarding experience of Jetpack guiding the users through a new onboarding process.
```

Comment:
The language here is so inconsistent. Some are imperative, while some are present continuous, while some are in past tense 😄